lib/modules/hideChildComments.js

```diff
@@ -42,7 +47,7 @@ module.go = () => {
 	const commentMenu = document.querySelector('ul.buttons');
 	if (commentMenu) {
 		commentMenu.appendChild(toggleButton);
-		const rootComments = document.querySelectorAll('div.commentarea > div.sitetable > div.thing > div.child > div.listing');
+		const rootComments = document.querySelectorAll('div.commentarea > div.sitetable > div.thing > div.child > div.listing, div.listing div.listing');
```

Comment:
This can probably be simplified to something like

```js
const rootComments = document.querySelectorAll('.commentarea .listing');
```

We don't need to keep the original, very precise selector now that all comments should have a "hide child comments" button.

And maybe it should be renamed, since it's not just the `rootComments` anymore.